loader.py

```diff
@@ -1,286 +0,0 @@
-#!/usr/bin/python
-
-##
-# loader.py
```

Comment:
Why was this deleted? You are going to need to put the file back to merge the pull request since GitHub is logging a conflict between your code and this repository.

Reply:
I deleted by mistake